File: java/program/Powersupply.java
class Powersupply extends Komponen {
    private int watt;

    public Powersupply() {
    }

    public Powersupply(int watt, String merk, String nama) {
        super(merk, nama);
        this.watt = watt;
    }

    public void setWatt(int watt) {
        this.watt = watt;
    }

    public int getWatt() {
        return this.watt;
    }
}

File: cpp/program/Powersupply.cpp
#pragma once
#include "Komponen.cpp"
#include <iostream>
#include <string>

class Powersupply : public Komponen {
  private:
    int watt;

  public:
    Powersupply() {}

    Powersupply(int watt, string merk, string nama)
        : Komponen(merk, nama) {
        this->watt = watt;
    }

    void setWatt(int watt) {
        this->watt = watt;
    }

    int getWatt() {
        return this->watt;
    }

    ~Powersupply() {}
};
